src/utils/cache.ts

```diff
@@ -70,8 +83,14 @@ export function blockTagParamIndex(payload: Payload): number | undefined {
   }
 }
 
-export function cacheTypeForPayload(payload: Payload): string {
-  switch (payload.method) {
+/**
+ * Return the cache type used for the given method.
+ *
+ * @param method - A JSON-RPC method.
+ * @returns The cache type to use for that method.
+ */
+export function cacheTypeForMethod(method?: string): string {
```

Comment:
Really curious why we decided to have this return `string` rather than `"perma" | "fork" | "block" | "never"`. Sigh...

Reply:
Yeah, missed opportunity. This is something I wanted to do as well - add an enum for this.

Reply:
Done here: https://github.com/MetaMask/eth-json-rpc-middleware/pull/184